seatunnel-engine/seatunnel-engine-core/src/main/java/org/apache/seatunnel/engine/core/classloader/DefaultClassLoaderService.java

```diff
@@ -140,7 +140,7 @@ int queryClassLoaderReferenceCount(long jobId, Collection<URL> jars) {
 
     /** Only for test */
     @VisibleForTesting
-    int queryClassLoaderCount() {
+    public int queryClassLoaderCount() {
```

Comment:
Why change this？

Reply:
> Why change this？

This method is used in the test case.

Reply:
If the test case in the same package with this class, the default scope is enough.

Reply:
> If the test case in the same package with this class, the default scope is enough.

It is an existing test case, which is in the zeta server package